docs/input-kafka.asciidoc

```diff
@@ -576,6 +577,20 @@ The schemas must follow a naming convention with the pattern <topic name>-value.
 Use either the Schema Registry config option or the
 <<plugins-{type}s-{plugin}-value_deserializer_class>> config option, but not both.
 
+[id="plugins-{type}s-{plugin}-validate_schema_registry"]
+===== `schema_registry_validation`
+
+  * Value can be either of: `auto`, `skip`
```

Comment:
👍🏼 I like the `auto`/`skip` here. It conveys the complexity a lot better than a `true`/`false` would, and similarly doesn't convey promises that we will validate it (if it isn't set, or we can't validate) or that we won't ever do so (e.g., when we start using it and it doesn't provide what we need).